### PR TITLE
Uml diagrams

### PR DESCRIPTION
**Pull request by BaranovMykola**
Added UML diagrams of general model struct.

### CODE REVIEW

File: UmlDiagram/Testing.uxf [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+--
+-QustionText: String
+/    -Image: BitmapImage/
+-ChoiseList: List&lt;Answer&gt;
+--
+/+AbstractOperation()/
+--
+Responsibilities
+-- Contains information about quastion text
```
Comment (OlyaOlga) [inline UmlDiagram/Testing.uxf]: question

File: UmlDiagram/Testing.uxf [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      <x>10</x>
+      <y>560</y>
+      <w>210</w>
+      <h>150</h>
+    </coordinates>
+    <panel_attributes>class Ticket
+
+--
+-Id: int
+-QuastionList: List&lt;Question&gt;
```
Comment (OlyaOlga) [inline UmlDiagram/Testing.uxf]: QuestionList

File: UmlDiagram/Testing.uxf [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    <coordinates>
+      <x>10</x>
+      <y>280</y>
+      <w>320</w>
+      <h>190</h>
+    </coordinates>
+    <panel_attributes>class Question
+
+--
+-QustionText: String
```
Comment (OlyaOlga) [inline UmlDiagram/Testing.uxf]: QuestionText

**Comment by BaranovMykola** [comment]
Fixed grammar errors

**Pull request merged**